.rubocop_todo.yml

```diff
@@ -127,12 +119,6 @@ RSpec/MultipleExpectations:
 RSpec/MultipleMemoizedHelpers:
   Max: 30
 
-# Offense count: 500
-# Configuration parameters: EnforcedStyle, IgnoreSharedExamples.
-# SupportedStyles: always, named_only
-RSpec/NamedSubject:
```

Comment:
This seems also unrelated?

Reply:
that was already applied by an earlier PR and the update code put it back in the file or missed it.

Reply:
@GarryHurleyJr What Jurre is asking, since this PR is about the removing `RSpec/BeforeAfterAll`, why we are removing this in this PR? This change has not been applied by any PR because [main](https://github.com/dependabot/dependabot-core/blob/main/.rubocop_todo.yml#L130-L134) still has this piece of code

Reply:
Reset and reapplied changes.  Not sure how this PR closed.